bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyColorUtils.java

```diff
@@ -29,10 +28,34 @@
  */
 @NonNullByDefault
 public class ShellyColorUtils {
+    OnOffType power = OnOffType.OFF;
+    String mode = "";
+
+    Integer red = 0;
+    Integer green = 0;
+    Integer blue = 0;
+    Integer white = 0;
```

Comment:
Any reason you can't use primitives instead?

Reply:
changed